test/Domain/LeanCode.CQRS.Tests/LoggingResponses/LoggingOperationResponsesTests.cs

```diff
@@ -0,0 +1,36 @@
+using Autofac;
+using LeanCode.CQRS.Execution;
+using Serilog;
+using Serilog.Events;
+using Xunit;
+
+namespace LeanCode.CQRS.Tests
+{
+    public class LoggingOperationResponsesTests : BaseCQRSTests
+    {
+        [Fact]
+        public async Task Correctly_log_operation_response_and_run_next()
+        {
+            Prepare();
+
+            var appCtx = new AppContext();
+            var fakeSink = new FakeSink();
+
+            var operationResponse = "Success!";
+            var nextFunctionResult = "Completed!";
```

Comment:
```suggestion
            const OperationResponse = "Success!";
            const NextFunctionResult = "Completed!";
```

plus let's move it to the class level.